ReactiveCocoaTests/Swift/ObjectiveCBridgingSpec.swift

```diff
@@ -58,26 +58,26 @@ class ObjectiveCBridgingSpec: QuickSpec {
 				var subscriptions = 0
 
 				let racSignal = RACSignal.createSignal { subscriber in
-					subscriber.sendNext(subscriptions)
-					subscriber.sendCompleted()
+					subscriber?.sendNext(subscriptions)
```

Comment:
Huh, how come this is `Optional` now? I imagine this used to be an implicitly unwrapped optional, in which case this should be unwrapped now, not ignoring it if it's `nil`.